App.tsx

```diff
@@ -1,10 +1,15 @@
+import AsyncStorage from "@react-native-async-storage/async-storage";
+import Parse from "parse";
 import React from "react";
 import { BottomTabNav } from "./src/components/bottom-tab-nav/bottom-tab-nav";
 import { TabInfo } from "./src/components/bottom-tab-nav/tab-details";
+import ParseServer from "./src/database/parse-server";
 
 // This disables the annoying warning messages in the simulator
 console.disableYellowBox = true;
 
+export let Database: Database = ParseServer.getInstance(Parse, AsyncStorage);
```

Comment:
Let me know if this is a good way to share the DB singleton across all the components. This seems ok, but I'm not sure if there's a more proper convention.

Reply:
this is how you do it. Is `ParseServer.getInstance(` like a get or create thing?/ if so resolve this comment :)

Reply:
this is perfect just needs to be a get or default deal which you did with the null checker